src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+Item {
+    id: item
+
+    property var model: NC.SyncStatusModel {}
```

Comment:
like in the other review, please declare it as a normal child of the root item and make it an alias property